src/actions/dashboard/detailAction.ts

```
@@ -59,17 +62,61 @@ export const action = async ({ request, params }: ActionFunctionArgs) => {
+      const commentId = Number(formData.get('commentId'));
+      const content = formData.get('content') as string;
+      await editComment(commentId, { content });
+      return { message: 'success' };
+    }
     case 'deleteComment':
+      {
+        const commentId = Number(formData.get('commentId'));
+        await deleteComment(commentId);
+        return { message: 'success' };
+      }
       break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)